main.go

```diff
@@ -187,6 +191,27 @@ func main() {
 	log.Printf("Attaching kprobes (via %s)...\n", msg)
 	ignored := 0
 	bar := pb.StartNew(len(funcs))
+
+	if flags.FilterTrackSkb {
+		skbLifetimeEndAt := []string{"kfree_skbmem"}
+		if !flags.FilterTrackSkbAcrossNamespaces {
+			skbLifetimeEndAt = append(skbLifetimeEndAt, "eth_type_trans")
```

Comment:
I don't know how to do it flexibly without hard coding `eth_type_trans` and `kfree_skbmem` - -